app/controllers/plans_controller.rb

```diff
@@ -1,3 +1,14 @@
+# Handle the plans model and the worksheet.
+#
+# Plans are typically created from the GoalsController, but are displayed,
+# edited, deleted, etc. from here. This is unusual for a REST app, but plans
+# are not really created from a plan sheet itself, but from the concept of a
+# set of Goals.
+#
+# Otherwise, updates, deletes, and shows work as expected.
+#
+# The Index operation (GET /plans/) displays the list of plans owned by the
+# user, which is a different template.
 class PlansController < ApplicationController
   before_action :authenticate_user!, only: %i[index]
```

Comment:
would it be worth documenting auth rules? we do have tests for these so maybe not all that important.

Reply:
I don't think so. I feel like any rubyist who comes along will be able to read them even easier than I could.

Reply:
Well... no, there is an unusual case here, so I'll document that.